Userland/Libraries/LibC/fcntl.cpp

```diff
@@ -104,6 +104,12 @@ int posix_fadvise(int fd, off_t offset, off_t len, int advice)
     return 0;
 }
 
+int posix_fallocate(int fd, off_t offset, off_t len)
+{
+    // posix_fallocate does not set errno
+    return static_cast<int>(syscall(SC_posix_fallocate, fd, &offset, &len));
```

Comment:
Not sure if this was already asked, but, why do we need to pass these by address? Is it because off_t is always 64-bit on our platform and we want to keep i386 around? Seems sad to pass word-size values by pointer on x86_64